src/index.html

```diff
@@ -1,17 +1,109 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label>
+        Surname:
+          <input type="text" name="Surname" 
+          autocomplete="off">
+        </label>
+        <label>
+        Name:
+          <input type="text" name="Name" 
+          autocomplete="off">
+        </label>
+        <label>
+        How old are You?
+          <input type="number" name="tentacles" 
+          min="1" max="100" 
+          value="12">
+        </label>
+        <label>
+        Full date of birth:
+          <input type="date" name="trip-start">
+        </label>
+        <label>
+        I accept the term of the agreement
+          <input type="checkbox" name="terms">
+        </label>
+      </fieldset>
+      <fieldset>
+        <legend>Registration</legend>
+        <label>
+        E-mail:
+          <input type="email" 
+          name="Email" placeholder="email@example.com">
+        </label>
+        <label>
+        Password:
+          <input type="password" name="password" 
+          minlength="5" maxlength="10">
+        </label>
+      </fieldset>
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <label>
+        Do you love cats?
+          <input type="radio" name="question love cats" 
+          value="m">
+        Yes
+          <input type="radio" name="question love cats"
+           value="f">
+        No
+        </label>
+        <label>
+        What is your favorite color?
+          <input type="color" name="head">
+        </label>
+        <label>
+        What time do you go to bed?
+          <input type="time" name="appt">
+        </label>
+        <label>
+        What are your favorite brand of cars?
+
+          <select name="car" multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+        <label>
+        How do you rate our work?
+          <input type="range" name="work"
+           min="1" max="10">
+        </label>
+      </fieldset>
+      <fieldset>
+        <legend>Additional info:</legend>
+        <label>
+        Comments:
+          <textarea name="Comments"></textarea>
+        </label>
+
+        <label>
+        Would you recommend us?
+          <select name="recommend us">
+            <option value="yes">yes</option>
+            <option value="no">no</option>
+          </select>
+        </label>
+      </fieldset>
+      <button type="submit">Submit</button>
+    </form>
   </body>
-</html>
+
+</html>
```

Comment:
```suggestion
</html>

```
add empty line at the end